src/lib.rs

```diff
@@ -103,21 +103,21 @@ pub enum AdcChannel {
 
 /// Set of options to control navigator's LEDs.
 #[derive(Debug, Clone, Copy)]
-pub enum LedColor {
+pub enum UserLed {
     /// Attached to LED_1 though GPIO 24.
```

Comment:
Can you add the color name for each one of these LEDs?

Reply:
and it's "through" there, not "though"

Reply:
did it.